websites/T/Twitch/presence.ts

```diff
@@ -467,7 +467,7 @@ presence.on("UpdateData", async () => {
 						profilePic =
 							document
 								.querySelector<HTMLImageElement>(
-									".tw-halo > .tw-aspect > .tw-avatar > .tw-image-avatar"
+									"#live-channel-stream-information > div > div > div.Layout-sc-1xcs6mc-0.dRGOOY > div > div.Layout-sc-1xcs6mc-0.iXRytg > div > div > div.Layout-sc-1xcs6mc-0.kpHsJz.avatar--t0iT1 > a > div > div.ScAvatar-sc-144b42z-0.dFBjte.tw-avatar > img"
```

Comment:
Is it possible to make this query less specific and with no computer-generated class names?

Eg:
```css
#live-channel-stream-information [class*=Layout][class*=avatar] .tw-avatar > img
```

Reply:
i just used the raw class names from twitch lmao. I can do some testing when i get a chance to see if i can make it a better query

Reply:
Any updates?